pkg/aspect/accessLogsManager_test.go

```diff
@@ -205,7 +205,8 @@ func TestAccessLoggerManager_ValidateConfig(t *testing.T) {
 
 	for idx, tt := range tests {
 		t.Run(fmt.Sprintf("[%d] %s", idx, tt.name), func(t *testing.T) {
-			if err := (&accessLogsManager{}).ValidateConfig(tt.cfg, expr.NewCEXLEvaluator(), tt.df); err != nil || tt.err != "" {
+			eval, _ := expr.NewCEXLEvaluator(1024)
```

Comment:
we choose 1024 a bunch of places here.  should we just make it a default size and have `expr.NewCEXLEvaluator()` use the default (for example, `return expr.NewCEXLEvalutaor(expr.DefaultCacheSize)`)?